app/mainAppWindow/index.js

```diff
@@ -53,20 +53,34 @@ exports.onAppReady = function onAppReady() {
 		window = null;
 	});
 
-	window.loadURL(config.url);
+	url = processArgs(process.argv)
+	window.loadURL( url ? url:config.url);
 
 	if (config.webDebug) {
 		window.openDevTools();
 	}
 }
 
-exports.onAppSecondInstance = function onAppSecondInstance(args) {
+exports.onAppSecondInstance = function onAppSecondInstance(event, args) {
 	console.log('second-instance started');
 	if (window) {
-		console.log('focusing on window');
-		if (window.isMinimized()) window.restore();
-		window.focus();
-		window.webContents.send('openUrl', args[0]);
+		event.preventDefault();
+		url = processArgs(args)
+		if (url) {
+			window.loadURL(url)
+		}
+	}
+}
+
+function processArgs(args) {
+	for (const arg of args) {
+		if (arg.startsWith('msteams:/l/meetup-join/')) {
+			console.log('showing on window');
+			window.show()
+			pathMeetup = arg.substring(8, arg.length)
+			url = 'https://teams.microsoft.com' + pathMeetup
```

Comment:
maybe use the config.url instead of the hardcoded value.

Reply:
I changed to use a variable `config.ul`